src/lazy/text/matched.rs

```diff
@@ -1259,14 +1396,11 @@ mod tests {
             (r"'''he''' '''llo'''", "hello"),
             (r#""😎🙂🙃""#, "😎🙂🙃"),
             (r"'''😎🙂''' '''🙃'''", "😎🙂🙃"),
-            // The below bytes are the UTF-8 encoding of Unicode code points: U+2764 U+FE0F
-            (r#""\xe2\x9d\xa4\xef\xb8\x8f""#, "❤️"),
-            (r"'''\xe2\x9d\xa4\xef\xb8\x8f'''", "❤️"),
```

Comment:
🗺️ In `clob`s, these escapes produce the correct UTF-8 data. In `string`s, they do not. Because the code points needed to represent this heart emoji are two bytes each, they cannot be represented via `\x` escapes, each of which can only represent 1-byte unicode scalar values.